### PR TITLE
Feature/qkm deploy #628

### PR DESCRIPTION
**Pull request by 6l20**
## PR Description

- Adds the ability to deploy new QKM
- Removes older KeyManager version
- Removes Vault initContainers
- Makes deployment parts optional

## Fixed Issue(s)

#628

## Documentation

- [ ] I thought about documentation and added the `documentation` label to this PR if updates are required.

### CODE REVIEW

File: values/qkm-postgres.yaml.gotmpl [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# QKM postgres config starts here
+
+postgresqlUsername: postgres
+postgresqlPassword: "1234567890"
```
[the remark this answers is not in the document]
Comment (6l20) [reply values/qkm-postgres.yaml.gotmpl]: Now added as an env var

File: environments/common.yaml.gotmpl [hunk 2]
```diff
@@ -1,26 +1,33 @@
 ---
 {{ $defaultNamespace := "orchestrate" }}
-{{ $tag := "v21.1.3" }}
+{{ $defaultOrchestrateChart := "consensys/orchestrate" }}
+{{ $defaultQkmChart := "consensys/quorumkeymanager" }}
+{{ $defaultOrchestrateChartVersion := "2.0.0" }}
+{{ $defaultQkmChartVersion := "1.1.5" }}
+{{ $defaultNamespace := "orchestrate" }}
+{{/* $tag := "v21.10.1-alpha.3" */}}
```
Comment (ggarri) [inline environments/common.yaml.gotmpl:8]: remove this commented line

File: environments/common.yaml.gotmpl [hunk 3]
```diff
@@ -79,10 +105,84 @@ postgresqlHA:
...
     repmgrPassword: such-secret
 
+qkm:
+  enabled: true
+  url: "http://quorum-key-manager-quorumkeymanager:8080"
+  proto: https
+  fullname: "quorumkeymanager"
+  orchestrate:
+    storeName: "eth-accounts"
+    apiKey: {{ env "QKM_API_KEY" | default "NmwyMCB3YXMgaGVyZQ==" }}
```
Comment (ggarri) [inline environments/common.yaml.gotmpl:115]: Does it make sense to have sensitive information such as API_KEY in default values?

File: environments/common.yaml.gotmpl [hunk 4]
```diff
@@ -79,10 +105,84 @@ postgresqlHA:
...
+  environmentSecrets: {}
+
+  postgresql:
+    tls:
+      enabled: false
+    host: {{ env "QKM_POSTGRES_HOST" | default "qkm-postgresql" }}
+    port: 5432
+    database: {{ env "QKM_POSTGRES_DB" | default "qkmDB" }}
+    username: {{ env "QKM_POSTGRES_USER" | default "qkm" }}
+    password: {{ env "QKM_POSTGRES_PWD" | default "qkmDBPwd" }}
```
Comment (ggarri) [inline environments/common.yaml.gotmpl:139]: Why do you have default pass in plain text?

File: environments/common.yaml.gotmpl (hunk 3 above)
Comment (6l20) [reply environments/common.yaml.gotmpl:117]: As it is stated in the README file every sensitive information should be identified and updated/removed by the end user. Having these values is just for informative purpose, so I see no problem with that. At least it provides info on the expected format.

File: environments/common.yaml.gotmpl (hunk 4 above)
Comment (6l20) [reply environments/common.yaml.gotmpl:141]: For the sake of simplicity it enables end users to have a demo solution running faster, and there again as stated in the README these values are under end user responsibility.

**Review by ggarri — APPROVED** [review]

**Pull request merged**